features/clipboard-unsanitized-formats.yml

```
@@ -0,0 +1,6 @@
+name: Paste unsanitized data
```

I worry a little about creating confusion with _paste_ (which I think of as whatever UI action that leads up to the `paste` event being fired). What do you think of one of these alternatives? (I don't have a strong feeling.)

```suggestion
name: Unsanitized clipboard formats
```

```suggestion
name: Unsanitized clipboard data
```

```suggestion
name: Reading unsanitized clipboard data
```

```suggestion
name: Unsanitized clipboard reads
```